### PR TITLE
Lesson 15

### PR DESCRIPTION
**Pull request by netscro**
Lesson 15

### CODE REVIEW

File: src/com/javabasic/lesson15/DegreeConverter.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        System.out.print("Please input degrees: ");
+        double degree = getDouble();
+        System.out.print("Please chose type to convert: \n");
+        System.out.println("Fahrenheit - 1");
+        System.out.println("Kelvin     - 2");
+        int degreeToConvert = getInt();
+        CreateDegree(degreeToConvert).convert(degree);
+    }
+
+    public static Converter CreateDegree(int degreeToConvert) {
```
Comment (kolodgeeva) [inline src/com/javabasic/lesson15/DegreeConverter.java]: CreateDegree -> createConverter

File: src/com/javabasic/lesson15/interfaces/Converter.java [hunk 2]
```diff
@@ -0,0 +1,7 @@
+package com.javabasic.lesson15.interfaces;
+
+public interface Converter {
+
+    void convert(double degree);
```
Comment (kolodgeeva) [inline src/com/javabasic/lesson15/interfaces/Converter.java]: It's better to return value from this method. Imagine you need to use it further in the code.

**Review by kolodgeeva — COMMENTED** [review]
Good understanding of interfaces and the factory method. Well done!

File: src/com/javabasic/lesson15/DegreeConverter.java (hunk 1 above)
Comment (netscro) [reply src/com/javabasic/lesson15/DegreeConverter.java]: done

File: src/com/javabasic/lesson15/interfaces/Converter.java (hunk 2 above)
Comment (netscro) [reply src/com/javabasic/lesson15/interfaces/Converter.java]: done

**Review by kolodgeeva — APPROVED** [review]

**Pull request merged**